### ISSUE TITLE
Remove "general" checks. Keep only accessibility checks

### ISSUE THREAD

**Issue by karlgroves** [karlgroves/diagnostic.css#11]
As the purpose of this stylesheet is specific to calling out accessibility issues, the general checks should be removed.
